test/screens/alerts/cache/filter_test.exs

```diff
@@ -3,6 +3,7 @@ defmodule Screens.Alerts.Cache.FilterTest do
 
   import Mox
```

Comment:
Not directly related to this change, but I'd suggest including `setup :verify_on_exit!` whenever we're using Mox, so tests fail if we set up an `expect` that's never used (alerting us that either something is wrong or the mock is redundant and can be removed). As-is it wouldn't have any effect on these tests, since `stub`s are never verified.

Reply:
🫡 here's the suggested change as its own PR https://github.com/mbta/screens/pull/2147